features/post-meta.feature

```diff
@@ -139,3 +139,95 @@ Feature: Manage post custom fields
       """
       My\New\Meta
       """
+
+  @pluck
+  Scenario: Multi-dimensional values can be plucked.
+    Given a WP install
+    And an input.json file:
+      """
+      {
+        "foo": "bar"
+      }
+      """
+    And I run `wp post meta set 1 meta-key --format=json < input.json`
+
+    When I run `wp post meta pluck 1 meta-key foo`
+    Then STDOUT should be:
+      """
+      bar
+      """
+
+  @pluck @pluck-deep
+  Scenario: Multi-dimensional values can be plucked at any depth.
+    Given a WP install
+    And an input.json file:
+      """
+      {
+        "foo": {
+          "bar": {
+            "baz": "some value"
+          }
+        },
+        "foo.com": {
+          "visitors": 999
+        }
+      }
+      """
+    And I run `wp post meta set 1 meta-key --format=json < input.json`
+
+    When I run `wp post meta pluck 1 meta-key foo.bar.baz`
```

Comment:
That sounds like a good idea. For `patch` then, should we use an associative argument for the new value or continue to use the last positional argument as the new value?